stable/unifi/templates/NOTES.txt

```diff
@@ -14,6 +14,6 @@
   echo http://$SERVICE_IP:{{ .Values.guiService.port }}
 {{- else if contains "ClusterIP" .Values.guiService.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "unifi.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
-  echo "Visit http://127.0.0.1:8080 to use your application"
-  kubectl port-forward $POD_NAME 8080:80
+  kubectl port-forward $POD_NAME 8443:8443
```

Comment:
I think it should be 

```suggestion
  kubectl port-forward $POD_NAME 8443:{.Values.guiService.port}
```

Reply:
Please correct and also sign the DCO before your contribution can be accepted. Thanks

Reply:
I updated the code to use .Values.guiService.port, thanks for the hint. Totally missed the variable for the port.